xbmc/cores/VideoPlayer/DVDClock.cpp

```diff
@@ -35,6 +36,7 @@ CDVDClock::CDVDClock()
   m_pauseClock = 0;
   m_bReset = true;
   m_paused = false;
+  m_speedAfterPause = DVD_PLAYSPEED_PAUSE;
```

Comment:
the value here seemed like the most likely to be a default value: https://github.com/xbmc/xbmc/blob/8405bbf242a73e7b88641c7db4a9808fd46a2f47/xbmc/cores/VideoPlayer/DVDClock.cpp#L113